### DISCUSSION TITLE
Terminal Voltage Calculation Problem

### DISCUSSION THREAD

**Discussion by JayYDG** [pybamm-team/liionpack#273] [General]
The terminal voltage is calculated by only one node. 

https://github.com/pybamm-team/liionpack/blob/cb38ed5113452d82ac4a63ec3ef34e62f61ad8ed/liionpack/solvers.py#L274

Should it be calculated by the difference between two nodes?
```
self.Terminal_Node1 = np.array(netlist[self.I_map].node1) 

self.Terminal_Node2 = np.array(netlist[self.I_map].node2)

self.V_terminal[step] = V_node[self.Terminal_Node1][0] - V_node[self.Terminal_Node2][0]
```

**Comment by TomTranter** [comment]
The second terminal node is always held at zero so the approaches are equivalent.
